### ISSUE TITLE
Do we want to include an "About" page?

### ISSUE THREAD

**Issue by ErnieAtLYD** [codeforamerica/mdc-feedback#501]
Or are we assuming that everyone will know exactly what this app does once it's live?

**Comment by mathias-gibson** [comment]
I think thats a great idea, even if people get directed here by finishing a survey, we could always provide more context in case they are curious. The questions is, what would go on there?
- brief explanation of fellowship/origin of the dashboard
- explanation of goal/purpose of the dashboard and new feedback system
- little blurb about code for america
- link to github repo

anything else?

**Comment by ErnieAtLYD** [comment]
Yeah, that works for me. Just write up the blurb - HTML or text, up to you - and I'll include it.
